test/parallel/test-cluster-master-kill.js

```diff
@@ -66,15 +59,17 @@ if (cluster.isWorker) {
     assert.equal(code, 0);
 
     // check worker process status
-    alive = isAlive(pid);
+    var pollWorker = function() {
+      alive = isAlive(pid);
+      if (alive) {
+        setTimeout(pollWorker, 500);
```

Comment:
i think you safely can set the timeout value lower, so we don´t slow down the test up to a half second.